### ISSUE TITLE
Updating fields on the event submission page

### ISSUE THREAD

**Issue by hudsonkelly83** [EclipseFdn/events.eclipse.org#200]
Hi Chris, 

I would like to put a project on your team's radar. There is no hard deadline we would like this completed, just whenever your team has time. 

We would like to update the fields on the event submission page. Below is what we would like the process to look like:

**Title** (Keep the same as it currently is)

**Event Type**

EclipseCon
Demo camps and Stammtisch
Eclipse Days & Hackathon
Working Group Event
Training Series
Other community event of interest
Unknown

(The edits here are removing virtual event from the list and changing 'Other interesting events' to 'Other community event of interest'

**Attendance Type** --> **Eclipse Foundation Participation**

(Please rename the attendance type field to Eclipse Foundation Participation, the options in the dropdown can stay the same and it should still result in color-coding the event on the event page)

**New Field:**  **Attendance Type**

(We would like to add a new field that adds an icon to the event listing, showing whether it is virtual, in person or a hybrid event. Whatever option is checked, an icon should appear over the event square that shows this. Please use these icons 

[2.zip](https://github.com/EclipseFdn/events.eclipse.org/files/6466228/2.zip)
[2 (2).zip](https://github.com/EclipseFdn/events.eclipse.org/files/6466234/2.2.zip)

A hybrid event should feature both icons, looking like this: 
![Untitled design (46)](https://user-images.githubusercontent.com/70344546/117978140-8c965880-b2ff-11eb-93c1-5e949b6c8ecd.png)

Website and registration fields can remain the same as they are, as can Date and Time, and the box for the text. 

For the Location box, please make this minimized by default if the attendance type is selected as virtual. 

Publishing information and everything else can remain the same. 

This may be outside the scope of this update, but for past events, perhaps we could have them automatically greyed out and drop below the current events, with maybe a header that distinguishes upcoming events from past events.

**Comment by autumnfound** [comment]
@chrisguindon I think this is for newsroom rather than here. Should we move this issue?

**Comment by chrisguindon** [comment]
I think it's fine here but we can create a new bug to request changes to the API since this will require a change to the newsroom and to this web app.

I was not planing for us to work on this now. This is something that we should look at later this summer.

**Comment by autumnfound** [comment]
Agreed, I wasn't thinking that either, but I wanted to be sure it was on your radar is all
